enterprise_access/apps/api/v1/views/content_assignments/assignments_admin.py

```diff
@@ -219,24 +221,27 @@ def cancel(self, request, *args, **kwargs):
         tags=[CONTENT_ASSIGNMENT_ADMIN_CRUD_API_TAG],
         summary='Cancel all assignments for the requested assignment configuration.',
         request=None,
+        filters=filters.LearnerContentAssignmentAdminFilter,
         responses={
             status.HTTP_202_ACCEPTED: None,
             status.HTTP_404_NOT_FOUND: None,
             status.HTTP_422_UNPROCESSABLE_ENTITY: None,
         },
     )
     @permission_required(CONTENT_ASSIGNMENT_ADMIN_WRITE_PERMISSION, fn=assignment_admin_permission_fn)
-    @action(detail=False, methods=['post'], url_path='cancel-all')
+    @action(detail=False, methods=['post'], url_path='cancel-all', pagination_class=None)
```

Comment:
The response isn't paginated, so make it super obvious to drf-spectacular that this is the case.